### PR TITLE
Hub: adds tooltip for badges and font for resource name

### PR DESCRIPTION
**Pull request by pratap0007**
this patch includes

- tooltip for resource type and support tier badge in resource card.
- `courier` and `monospace` font for resource name.
- limits on number of tags in resource card.

Signed-off-by: Shiv Verma <shverma@redhat.com>

# Changes

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/experimental/blob/master/CONTRIBUTING.md)
for more details._

### CODE REVIEW

File: hub/ui-new/.eslintrc [hunk 1]
```diff
@@ -0,0 +1,42 @@
+{
```
Comment (sthaha) [inline hub/ui-new/.eslintrc:1]: why is this needed as a part of this patch?

File: hub/ui-new/src/components/basic-detail/BasicDetail.tsx [hunk 2]
```diff
@@ -166,10 +164,13 @@ const BasicDetail: React.FC<BasicDetailProp> = (props: any) => {
...
 
-        displayName = data.dsiplayName === ' ' ? data.resource.name :
-          displayName.replace(/(^\w|\s+\w){1}/g, ((str) => {
-            return str.toUpperCase();
-          }));
+        resourceName = data.displayName === '' ?
+          <span style={{fontFamily: 'courier, monospace'}}>
+            {data.resource.name}</span> :
+          <p >{data.displayName}
+            {<span style={{fontFamily: 'courier, monospace'}}>
```
Comment (sthaha) [inline hub/ui-new/src/components/basic-detail/BasicDetail.tsx:171]: why do we display both displayName and name?

File: hub/ui-new/.eslintrc (hunk 1 above)
Comment (pratap0007) [reply hub/ui-new/.eslintrc:1]: I did it for ui-new because it helped in solving eslint error for ui-new.
